adaptors/dummy/deploy/e2e/usability/suite_test.go

```diff
@@ -0,0 +1,109 @@
+package usability_test
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	edgev1alpha1 "github.com/rancher/octopus/api/v1alpha1"
+	dummyv1alpha1 "github.com/rancher/octopus/adaptors/dummy/api/v1alpha1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	clientsetscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest/printer"
+
+	"github.com/rancher/octopus/pkg/brain"
+	"github.com/rancher/octopus/pkg/limb"
+	"github.com/rancher/octopus/test/framework"
+)
+
+var (
+	testCtx       context.Context
+	testCtxCancel context.CancelFunc
+	testCurrDir   string
+	testRootDir   string
+	testEnv       *envtest.Environment
+
+	k8sCfg *rest.Config
+	k8sCli client.Client
+
+	deviceLinkNamespace = "default"
+	deviceLinkName      = "test-fan-"
+	deviceLink          edgev1alpha1.DeviceLink
+	nodeName            string
+)
+
+func TestOctopus(t *testing.T) {
+	defer GinkgoRecover()
+
+	RegisterFailHandler(Fail)
+
+	RunSpecsWithDefaultAndCustomReporters(t,
+		"dummy devicelink suite",
```

Comment:
We should keep the description of the suite as the same as the testing domain, please change to `usability suite`.

Reply:
updated